### ISSUE TITLE
bug behavior summary and comment

### ISSUE THREAD

**Issue by Jayodin** [ValveSoftware/Dota2-Gameplay#13390]
### Description

hi i have big problem my comment 3weak is lock and never change just in 912 stay and i give a lot comment bot never change
and my behavior summary just going down and never go up 25day i have this bug and never repair pls help me.
i play good and give comment i just 1 star to go immortal but this bug destroyed my accunt and i dump to divine 1 pls help to fix this bug ,
its so bad dota 2 dont have private support and its for this big game so bad how player can speak with support dota2 i ticket to steam and other support but they just say go say to other supps
pls heeeelp i tired
id dota2 : 232052643

### Example Match ID (and possibly Timestamp)

232052643

**Comment by BledNavalny** [comment]
Yeah, dota team is small stupid group of people with no feedback.
Feel sorry for you. I might be in the same position, but i gained immortal 2 months before september where they apologised for new system. But they dont learn and did this again.

That time i played at 1500 beh with voice chat and felt good. Nobody ruined game on purpose.

But now they put all people in gulag. This might be a joke from them, hat most people of dota and cs go are speak russian

( i had 4500 and raised to 6000 but after only minus)

**Comment by marcelmaster** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/13340
same here , upvote please ( thumbs up )
